### PR TITLE
MGMT-3335 - Add None Platform support 

### PR DESCRIPTION
**Pull request by eliorerz**
…load_balancer target with test target
TerraformController now take the PLATFORM environment variable and copy the none terraform files accordingly 

https://issues.redhat.com/browse/MGMT-3335

### CODE REVIEW

File: discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py [hunk 1]
```diff
@@ -32,7 +33,7 @@ def _create_tf_folder(self):
         tf_folder = utils.get_tf_folder(self.cluster_name)
         logging.info("Creating %s as terraform folder", tf_folder)
         utils.recreate_folder(tf_folder)
-        utils.copy_template_tree(tf_folder)
+        utils.copy_template_tree(tf_folder, self.platform == consts.Platforms.NONE)
```
Comment (tsorya) [inline discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py:36]: lets add none_platform_mode=self.platform == consts.Platforms.NONE

Comment (eliorerz) [reply discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py]: Done

File: discovery-infra/test_infra/helper_classes/cluster.py [hunk 2]
```diff
@@ -532,8 +534,12 @@ def prepare_for_install(
             ssh_key=env_variables['ssh_public_key'],
             nodes_count=env_variables['num_nodes'],
             vip_dhcp_allocation=env_variables['vip_dhcp_allocation'],
-            download_image=True
+            download_image=True,
+            platform=env_variables['platform']
     ):
+        if platform == consts.Platforms.NONE:
```
Comment (ori-amizur) [inline discovery-infra/test_infra/helper_classes/cluster.py:540]: IPv4 only

File: Makefile [hunk 3]
```diff
@@ -398,7 +398,7 @@ lint:
 _lint:
 	pre-commit run --all-files
 
-test:
+test: start_load_balancer
```
[the remark this answers is not in the document]
Comment (ori-amizur) [reply Makefile:401]: It has to be invoked not from a container

File: discovery-infra/test_infra/helper_classes/cluster.py (hunk 2 above)
Comment (eliorerz) [reply discovery-infra/test_infra/helper_classes/cluster.py]: Done

File: Makefile (hunk 3 above)
[the remark this answers is not in the document]
Comment (tsorya) [reply Makefile:401]: It has to be invoked not from container only for the case when we to leave it after test , no?

**Pull request merged**